blocks/library/cover-image/index.js

```diff
@@ -177,3 +189,9 @@ registerBlockType( 'core/cover-image', {
 		);
 	},
 } );
+
+function dimRatioToClass( ratio ) {
+	return ( ratio === 0 || ratio === 50 )
+		? null
+		: 'has-background-dim-' + Math.round( ratio / 10 );
```

Comment:
One more thing: Do we need the division? I might think `.has-background-dim-50` is clearer at a glance than `.has-background-dim-5`

Reply:
The classes reflected the allowed granularity, but I can change that. Maybe it'll make it more future-proof.

Reply:
Actually, the division might still be needed, but the computation becomes:

`10 * Math.round( ratio / 10 )`

meaning that 35…44 -> 40, etc.